src/ansys/aedt/core/extensions/project/configure_layout.py

```diff
@@ -379,12 +384,9 @@ def export_template_config():
             content = file.read()
 
         try:
-            dst_path = download_file(source="edb/ANSYS_SVP_V1_1.aedb", local_path=ExtensionDataLoad.working_directory)
+            download_file(source="edb/ANSYS_SVP_V1_1.aedb", local_path=ExtensionDataLoad.working_directory)
         except Exception:  # pragma: no cover
-            dst_path = ""
-
-        if dst_path:
-            content = re.sub(r'(layout_file\s*=\s*)".+?"', lambda m: m.group(1) + f"'{dst_path}'", content)
+            print("Failed to download example board.")
```

Comment:
```suggestion
            raise AEDTRuntimeError("Failed to download example board.")
```
This might need an import

Reply:
It is not intended to break here. I use msg instead.

Reply:
When used through the UI, raisins an errer opens a dialogue Windows and doesnt break AFAIK

Reply:
The logic is like this
if there is internet connection, it copies the config files and downloads the example Edb.
if there is no internet connection, it copies the config files but skip downloading the example Edb. 

I refactored to add message instead of raising error or print.